src/docs/reference/core/Razor/README.md

```diff
@@ -44,6 +44,7 @@ Many extensions methods are available in Razor with `@Orchard`.
 If you want to use an extension method in a view, you can inject an `IOrchardHelper` named `Orchard` at the top of your file:
 
 ```csharp
+@using OrchardCore
 @inject OrchardCore.IOrchardHelper Orchard
```

Comment:
No need for `OrchardCore.` here then, no?

Reply:
I did end up adding mine to the `_ViewImports.cshtml` instead of adding it here. Should I remove this from the usage examples and just write a comment that you might need to add the using statement?

Reply:
I mean, the `using` is good, I'm talking about having`@inject IOrchardHelper Orchard` instead of `@inject OrchardCore.IOrchardHelper Orchard`, since you already import the namespace.

But otherwise yeah, you could clarify with e.g.:

```csharp
// You can also put this into a _ViewImports.cshtml file so it's applied to all templates.
@using OrchardCore
```

Reply:
Oh yeah, I misunderstood. I will remove the `OrchardCore.` and add the comment.